relevanceai/dataset_api/dataset.py

```diff
@@ -2,6 +2,7 @@
 Pandas like dataset API
 """
 import warnings
+from cv2 import sort
```

Comment:
Do we need this?

Reply:
Im not sure why this was added, I think by accidental tab complete